dell/src/main/java/org/apache/iceberg/dell/DellClientFactory.java

```diff
@@ -20,9 +20,10 @@
 package org.apache.iceberg.dell;
 
 import com.emc.object.s3.S3Client;
+import java.io.Serializable;
 import java.util.Map;
 
-public interface DellClientFactory {
+public interface DellClientFactory extends Serializable {
```

Comment:
Can you move code changes out of a documentation PR? These should be in a code PR for people that cherry-pick commits.